### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by janmarkevin**

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -1,18 +1,52 @@
...
-    <link rel="stylesheet" href="css/main.min.css" />
+    <title>Seisaku</title>
+    <meta name="description" content="Seisaku" />
+    <link rel="stylesheet" href="css/style.css" />
     <!--[if lt IE 9]><script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script><script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script> <![endif]-->
 </head>
-
 <body>
-    <h1>Hello World!</h1>
+<header class="header">
```
Comment (sayeaah) [inline dist/index.html]: kindly fix indention of whole header block po ~

File: dist/index.html [hunk 2]
```diff
@@ -1,18 +1,52 @@
...
+    <div class="header__wrapper">
+        <div class="header__logo-container">
+            <div class="logo">
+                <span class="logo__text">ホームページ制作は豊富な実績のアップワード</span>
+                <a class="logo__img-wrapper">
```
Comment (sayeaah) [inline dist/index.html]: Kindly add `href` value

File: dist/index.html [hunk 3]
```diff
@@ -1,18 +1,52 @@
...
+                    <img src="./img/logo.png" alt="logo" class="logo__img">
+                </a>
+            </div>
+        </div>
+        <div class="header__links">
+            <div class="header__btn">
+                <div class="btn">
```
Comment (sayeaah) [inline dist/index.html]: Kindly use `<button>` element

File: dist/index.html [hunk 4]
```diff
@@ -1,18 +1,52 @@
...
+                    <img src="./img/tele.png" alt="Telephone" class="btn__icon btn__icon--header-tele">
+                    <span class="btn__text btn__text--larger">03-5826-4652</span><br>
```
Comment (sayeaah) [inline dist/index.html]: Kindly refrain from using `<br>`.

File: dist/index.html [hunk 5]
```diff
@@ -1,18 +1,52 @@
...
+                    <span class="btn__text btn__text--small">
+                    平日受付時間 午前9:00～午後18:00</span>
```
Comment (sayeaah) [inline dist/index.html]: Kindly fix indention to 
```
<span>
   {{ text }}
</span>
```

File: dist/index.html [hunk 6]
```diff
@@ -1,18 +1,52 @@
...
+                </div>
+                <div class="btn btn--header-mail">
```
Comment (sayeaah) [inline dist/index.html]: Kindly use <button> element

File: dist/index.html [hunk 7]
```diff
@@ -1,18 +1,52 @@
...
+                    <img src="./img/mail.png" alt="main" class="btn__icon btn__icon btn__icon--header-mail">
+                    <span class="btn__text">メールでお問い合わせ</span>
+                </div>
+            </div>
+            <div class="header__nav-container">
+                <div class="nav">
```
Comment (sayeaah) [inline dist/index.html]: Kindly use <nav>

File: dist/index.html [hunk 8]
```diff
@@ -1,18 +1,52 @@
...
+                    <ul class="nav__list">
+                        <li class="nav__item"><a href="#" class="nav__link"><img src="./img/Asset3.png" alt="Home Icon"></a></li>
```
Comment (sayeaah) [inline dist/index.html]: Kindly put links on a separate line

File: src/scss/components/_button.scss [hunk 9]
```diff
@@ -0,0 +1,39 @@
+.btn{
```
Comment (sayeaah) [inline src/scss/components/_button.scss]: Kindly add space before the {

File: src/scss/components/_button.scss [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+  padding: .5rem 1.5rem .5rem 4rem;
```
Comment (sayeaah) [inline src/scss/components/_button.scss]: For values that are not font-size, kindly use `px` and also for decimal values. Kindly put 0 first.
 Example: 0.5px

Comment (sayeaah) [reply src/scss/components/_button.scss]: Please apply this comment to all scss files 🙇

File: src/scss/components/_header.scss [hunk 11]
```diff
@@ -0,0 +1,24 @@
+.header{
+    background: rgba($white-color,.8);
```
Comment (sayeaah) [inline src/scss/components/_header.scss]: Add space after `,`

Comment (sayeaah) [reply src/scss/components/_header.scss]: Kindly apply this to all scss files as well

File: src/scss/settings/_settings.scss [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+@font-face {
+    font-family: 'yugothil';
+    src: url('/dist/fonts/yugothil.ttf') format(truetype);
+}
+@font-face {
+    font-family: 'yumin';
+    src: url('/dist/fonts/yumin.ttf') format(truetype);
+}
+// colors
+$primary-color: #CD282D;
```
Comment (sayeaah) [inline src/scss/settings/_settings.scss]: Kindly use lowercase for color hex

File: src/scss/settings/_settings.scss [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+$secondary-color: #4D4D4D;
+$light-grey: #666666;
```
Comment (sayeaah) [inline src/scss/settings/_settings.scss]: Kindly use shorthand for repeating hex.
Example:
```
$light-grey: #666;
$white: #fff;
$black: #000;
```

**Review by sayeaah — COMMENTED** [review]
@janmarkevin Added comments. Kindly check po ~

File: src/scss/components/_button.scss (hunk 10 above)
Comment (janmarkevin) [reply src/scss/components/_button.scss]: hahahha okay po sorry hahaahha

File: dist/index.html [hunk 14]
```diff
@@ -10,43 +10,54 @@
...
+                    <button class="btn btn--header-mail">
+                        <img src="img/mail.png" alt="main" class="btn__icon btn__icon btn__icon--header-mail">
+                        <span class="btn__text">メールでお問い合わせ</span>
+                    </button>
+                </div>
+                <div class="header__nav-container">
+                    <nav class="nav">
+                        <ul class="nav__list">
+                            <li class="nav__item">
+                                <a href="#" class="nav__link"><img src="img/Asset3.png" alt="Home Icon"></a>
```
Comment (sayeaah) [inline dist/index.html]: Kindly put img in separate line

**Pull request merged**